ISISScientificComputing/autoreduce
# Integrate validate data ready

### Summary of work
* Add validation step into run_detection code
* Add test cases

### How to test your work
* This needs to be tested on the ingestion node.
  * Apply the changes to the node
  * Change the `last_run.csv` to point to a previous (maybe a run less than current)
  * Ensure the ingestion windows task is scheduled
  * Ensure the Message is sent successfully to the queues




**Before merging ensure the release notes have been updated**
